index.js

```diff
@@ -178,7 +181,7 @@ AFRAME.registerComponent('teleport-controls', {
       if (this.numActivePoints > this.data.curveNumberPoints){
         this.numActivePoints = this.data.curveNumberPoints;
       }
-
+      this.numActivePoints = Math.round(this.numActivePoints);
```

Comment:
In the previous version of the code, the name "this.numActivePoints" was poorly chosen, since it was not a rounded number. However, it was good that it wasn't a round number because that allowed the curve to be drawn smoothly (rather than in chunks), since the the time parameter passed in to `parabolicCurve` took this unrounded number into account. See https://github.com/MozillaReality/aframe-teleport-controls/pull/5/files#diff-168726dbe96b3ce427e7fedce31bb0bcL213
I think what you have here is clearer, but we should still draw the curve smoothly by setting its last point to be between two of the points we use as the curve gets longer.

Reply:
Unrelated to my comment above, the line should not reach the hitEntity quite so fast when I am teleporting near to me. Right now short teleports happen quickly and long teleports take a long time. We want to balance it out so that short-distance teleports and long-distance teleports take the same amount of time. This will require a combination of the time since the teleport began with the information from the raycasting step to decide how much of the curve to draw on this tick.